lib/transfer_jobs/railtie.rb

```diff
@@ -0,0 +1,6 @@
+class TransferJobs::Railtie < Rails::Railtie
```

Comment:
The `shopify-tracing` gem does a `require 'rails'` here: https://github.com/Shopify/shopify-tracing/blob/master/lib/shopify/tracing/adapters/rails_adapter.rb#L3

IDK if that's necessary. If it is, then add it here. If it isn't, then open an issue to remove it from `shopify-tracing`.

It kinda feels like it shouldn't be necessary, since this file should only be required if `Rails` is defined, and that should only happen if someone already `require`d `'rails'`.

Reply:
I've tested this out in the `avalara-app` and it worked. I think that this isn't required since in the gem file we already check if `Rails` has been required.